src/main/java/net/pms/encoders/FFmpegAudio.java

```diff
@@ -211,15 +211,25 @@ public ProcessWrapper launchTranscode(
 		} else { // default: LPCM
 			cmdList.add("-f");
 			cmdList.add("s16be");
+			cmdList.add("-sample_fmt");
+			cmdList.add("s16");
 		}
 
 		if (configuration.isAudioResample()) {
 			if (params.mediaRenderer.isTranscodeAudioTo441()) {
 				cmdList.add("-ar");
 				cmdList.add("44100");
+				cmdList.add("-ac");
+				cmdList.add("2");
+				cmdList.add("-ab");
+				cmdList.add("1411200");
```

Comment:
You really don't want to hardcode such non-standard values for bitrate - leave alone standard values.